ontology/yaml/resources/GATEWAYS/entity_types/GW.yaml

```diff
@@ -747,13 +747,13 @@ LROC400:
   - green_scene_dimmer_percentage_command_40
   - blue_scene_dimmer_percentage_command_40
   - startup_dimmer_percentage_setpoint_40
-  - power_command
```

Comment:
@tasodorff this is a breaking change?

Reply:
I'm not sure what you mean by "breaking change" I had to change the field name because the subfield "power" is related to the units "watts", "kilowatts" ect. In our case this field represents an actuator or KNX driven switch with the states "on" and "off". Therefore I had to change the name of the field.

Reply:
I see, thank you